packages/schematics/update/update/npm.ts

```diff
@@ -95,6 +96,9 @@ export function getNpmPackageJson(
               http: options[0],
               https: options[1],
             },
+            ssl: {
+              strict: options[2],
```

Comment:
This will give you the string value instead of the boolean. options[2] === 'true' would be more appropriate.